exercises/practice/raindrops/.meta/config.json

```
@@ -1,6 +1,6 @@
 {
   "authors": [
-    "axtens"
+    "Caraciola"
```

You're still an author, but you can have multiple:

```suggestion
    "axtens",
    "Caraciola"
```